blog/2026-02-25-dev-update-february-2026.md

```
@@ -0,0 +1,213 @@
+      ws: ${ENVIO_WS_ENDPOINT}
+      for: live
+```
+
+
+
+### 🚨Breaking: rpc_config Removed
+
+`rpc_config` has been removed in favour of **<code>rpc</code>**.
+
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)